src/NodaTime.Web/Markdown/1.0.x/concepts.md

```diff
@@ -144,18 +144,20 @@ change occurs - usually for daylight saving changes.
 Most of the time when you use a `DateTimeZone` you won't need
 worry about that - the main purpose is usually to convert between a
 [`ZonedDateTime`][ZonedDateTime] and a [`LocalDateTime`][LocalDateTime]
-- where the names mean exactly what you expect them to. There's a slight
+, where the names mean exactly what you expect them to. There's a slight
 twist to this: converting from an `Instant` or a `ZonedDateTime` to a
 `LocalDateTime` is unambiguous; at any point in time, all the (accurate)
 clocks in a particular time zone will show the same time... but the
 reverse isn't true. Any one local time can map to:
 
 - A single instant in time: this is the case for almost all the time.
+
```

Comment:
There shouldn't be any need for this (or the next change). These are deliberately list bullets.